tests/Makefile.tests_common

```diff
@@ -12,9 +12,6 @@ ifneq (,$(wildcard $(CURDIR)/tests*/.))
   endif
 endif
 
-ifneq (,$(filter tests_driver_%,$(APPLICATION)))
```

Comment:
It is no longer needed to check if the given tests is a `driver` test. This is now done in the `tests/drivers/Makefile.drivers_common`